<launch>
 
  
<node pkg="map_store" type="map_manager" name="map_manager" output="screen" />
  
</launch>
